docs/user-guide/meta-models.md

```diff
@@ -58,9 +58,7 @@ Wall time: 917 ms
 
 ## Grouped Prediction
 
-<p align="center">
-  <img src="../_static/meta-models/grouped-model.png" />
-</p>
+![grouped-model](../_static/meta-models/grouped-model.png)
```

Comment:
The only real thing to comes to mind is to mention the classifier/regressor objects in the docs here maybe?

Reply:
Sounds reasonable